src/bindings/python/src/pyopenvino/utils/utils.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <openvino/core/any.hpp>
+
+ov::Any convert_from_py_object(const pybind11::object& py_obj) {
```

Comment:
Could you rename it to something like `py_object_to_any` so that the name indicates the return type too?

Reply:
done